iotics/lib/identity/crypto/identity.py

```diff
@@ -37,4 +37,5 @@ def make_identifier(public_bytes: bytes) -> str:
     checksum = bytearray.fromhex(cl2.hexdigest())[:4]
 
     return IDENTIFIER_PREFIX + base58.b58encode(bytes([IDENTIFIER_METHOD, IDENTIFIER_VERSION, IDENTIFIER_PAD])
-                                                + pk_digest + checksum).decode('ascii')
+                                                + pk_digest + checksum,
+                                                alphabet=base58.BITCOIN_ALPHABET).decode('ascii')
```

Comment:
the use of this alphabet is fundamental - how do other impl of the spec would know about this? 
would all of them need to use the same "bitcoin_alphabet" ??

Reply:
base58 was written for bitcoin so the default is "bitcoin alphabet". I only specify here in case the default in the upstream library changes and to make the code clearer to read.  I'm happy to revert this override if you prefer.